### PR TITLE
Enhance FAQS Section and Added Forums | Issue #129

### LINKED ISSUE #129: [Feature Request]: Addition of Users Forum in FAQs Page and Enhancement of FAQs Page | User Can also Post
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

I want to enhance the FAQs Page and Add Forums section with Following Functionalities:-

1. Enhance the FAQs Page.
2. Forum Section with Listing of User's Posts in a Grid of Cards with Flip Effect on Cards.
4. User can also Post.
5. Sorting Functionality.
6. Responsive UI.

Please assign this to me under GSSOC Ext.

### Use Case

.

### Benefits

_No response_

### Priority

High

### Record

- [X] I agree to follow this project's Code of Conduct
- [ ] I'm a WOB contributor
- [X] I want to work on this issue
- [X] I'm willing to provide further clarification or assistance if needed.

### PR DESCRIPTION
**Pull request by Shariq2003**
fix #129 

Please View Video for a Better understanding of the Changes

https://github.com/user-attachments/assets/5796a6eb-003f-444a-b671-f5e46996859e

### CODE REVIEW

**Comment by Shariq2003** [comment]
@4darsh-Dev please review and merge

**Comment by panwar8279** [comment]
There are some deployment error, please create a new PR for this.

**Comment by Shariq2003** [comment]
Please give me ss of error

File: frontend/src/pages/FAQPage.jsx
Comment (4darsh-Dev) [inline frontend/src/pages/FAQPage.jsx:201]: ## dont use footer 
This will create footer container twice, 
see preview : https://decentrade-git-fork-shariq2003-e-3d36b2-adarsh-mauryas-projects.vercel.app

**Review by 4darsh-Dev — COMMENTED** [review]
nice ui @Shariq2003 
## A minor changes is required fix it

**Review by 4darsh-Dev — CHANGES_REQUESTED** [review]
do it asap

**Comment by Shariq2003** [comment]
> do it asap

okay sir fixing

**Comment by Shariq2003** [comment]
> do it asap

@4darsh-Dev sir, please review and merge

**Comment by Shariq2003** [comment]
@MastanSayyad sir please review

**Comment by MastanSayyad** [comment]
@Shariq2003 PA has reviewed the changes few hours ago, give him some time

**Comment by MastanSayyad** [comment]
as there are some deployment errors I cant merge this unless PA himself does!

**Comment by Shariq2003** [comment]
> as there are some deployment errors I cant merge this unless PA himself does!

I have resolved.

**Comment by 4darsh-Dev** [comment]
Gentle Reminder 🚨🚨 
 GSSoC-Ext 24 is going to complete in 2 Days. '
 Complete your assigned Issues and PR reviews before time to get Points on Leaderboard. '
 It was a great experience working with you all, Don't forget to ⭐ star the Repo (only 12 with 62 forks 💔) . 
 Thanks💗 for your valuable contributions!

**Pull request merged**
